hr_attendance_ip_check/models/hr_employee.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024 Janik von Rotz <janik.vonrotz@mint-system.ch>
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import ipaddress
+import logging
+
+from odoo import _, exceptions, fields, models
+from odoo.http import request
+
+_logger = logging.getLogger(__name__)
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    attendance_cidr_ids = fields.Many2many("attendance.cidr", ondelete="restrict")
+
+    def _attendance_ip_check(self):
```

Comment:
When called this method is doing several calls on `self` but it can be a recordset and will lead to failures

Reply:
Branch updated, please check again